backend/debug/dwarf/dwarf_high/dwarf_attribute_helpers.ml

```diff
@@ -131,6 +138,24 @@ let create_decl_column column =
   let column = Uint64.of_nonnegative_int_exn column in
   AV.create spec (V.uleb128 ~comment:"column number" column)
 
+let create_call_file file =
+  let spec = AS.create Call_file Udata in
+  let file = Uint64.of_nonnegative_int_exn file in
+  AV.create spec (V.uleb128 ~comment:"file number" file)
+
+let create_call_line line =
+  let spec = AS.create Call_line Udata in
+  let line = Uint64.of_nonnegative_int_exn line in
+  AV.create spec (V.uleb128 ~comment:"line number" line)
+
+let create_call_column column =
+  (* CR mshinwell: Check whether "call_file", "call_line" and "call_column" are
+     indeed supported pre DWARF 5. The gdb manual suggests that they've been
+     around since DWARF 2. *)
+  let spec = AS.create Call_column Udata in
+  let column = Uint64.of_nonnegative_int_exn column in
+  AV.create spec (V.uleb128 ~comment:"column number" column)
```

Comment:
(These three functions could be factored out,
but I am not positive it is worth the effort.)

Reply:
Yeah, not sure it's worth it.